src/main/java/monitoringsystemturbo/presenter/ConfirmExportPresenter.java

```diff
@@ -62,13 +65,13 @@ public void setPrimaryStage(Stage primaryStage) {
 
     private void setCheckboxes() {
         boolean areAllCheckboxesSelected = areAllCheckboxesSelected();
```

Comment:
there is no need to use this variable. It has the same name as a method, so it just an unnecessary line